src/objective-c/tests/PerfTests/PerfTests.m

```diff
@@ -181,7 +181,7 @@ - (void)testPingPongRPCWithV2API {
   [self pingPongV2APIWithRequest:request numMessages:1000 options:options];
 
   [self measureBlock:^{
-    [self pingPongV2APIWithRequest:request numMessages:10000 options:options];
+    [self pingPongV2APIWithRequest:request numMessages:1000 options:options];
```

Comment:
Could you explain a bit why we are reducing the number? Maybe in the PR description. For record.

Reply:
It's to reduce test run time. Takes too long with 10k messages, and we don't really get any additional benefit with more messages. I've updated the PR description.